app/assets/stylesheets/refills/_hero.scss

```diff
@@ -0,0 +1,30 @@
+.hero {
+  $large-screen: 50em !default;
```

Comment:
What does the `!default` do?

Reply:
From the SASS documentation:
```You can assign to variables if they aren’t already assigned by adding the !default flag to the end of the value. This means that if the variable has already been assigned to, it won’t be re-assigned, but if it doesn’t have a value yet, it will be given one.```

Reply:
Haven't looked too hard at the refills snippets yet, so not sure if this is a feature I will use.